### ISSUE TITLE
Implement the Debug menu from ImageJ 1.x

### ISSUE THREAD

**Issue by ctrueden** [scijava/script-editor#5]
The ImageJ 1.x macro editor has [a Debug menu with some useful debugging features](http://imagej.net/developer/macro/macros.html#debugger).

It would be great if the Script Editor had these features as well.

See these discussions:
* http://forum.imagej.net/t/debug-menu-missing-in-fiji-editor/3094
* http://imagej.1557.x6.nabble.com/Debugging-macros-in-Fiji-td5010227.html

**Comment by esiefker** [comment]
Yes, please. I struggle with the lack of debug every time I use Fiji.   

The copy and paste trick to access the old editor doesn't work for me.  Well, it operates but it throws errors that don't happen when I run the macro so it's not useful for debugging.  I'm stuck with selecting code line by line and hitting ctrl-shift-R.  Please make this better.

**Comment by ctrueden** [comment]
@esiefker Thanks for the vote. Unfortunately, I have [too many other urgent priorities](https://imagej.net/User:Rueden#Primary_projects) to work on the Script Editor right now, and there is unfortunately no one else taking care of this component besides me. However, if you or your group have any programming resources/capabilities which can be directed toward this mini-project, I would be delighted to provide guidance and support toward getting the feature implemented. Otherwise, I will get to it eventually, since this _is_ a feature which I agree would be really useful—it's just going to be a few months at least if it falls on my personally to do it.
